idea/resources-en/intentionDescriptions/HLImportAllMembersIntention/after.kt.template

```diff
@@ -0,0 +1,7 @@
+import javax.swing.SwingUtilities
+<spot>import javax.swing.SwingUtilities.*</spot>
+
+fun foo() {
+    SwingUtilities.invokeLater { }
+    SwingUtilities.invokeAndWait()
```

Comment:
It should probably be like this:

```suggestion
    invokeLater { }
    invokeAndWait()
```

Reply:
Good point! Let me update this.

Reply:
@tgeng I've missed that change and merged without it. Do you want me to extract that change from your commit and merge it?

Reply:
Sure, feel free to do whatever that's easier for you!